### PR TITLE
adding virtualenv version check

### PR DESCRIPTION
**Pull request by abhay-agarwal**

### CODE REVIEW

File: dcos-cli/install-dcos-cli.sh [hunk 1]
```diff
@@ -20,6 +20,11 @@ MARATHON_HOST=${ARGS[2]}
 MARATHON_PORT=${ARGS[3]:-8080}
 UNIVERSE_URI="https://github.com/mesosphere/universe/archive/ea.zip"
 
+command -v virtualenv >/dev/null 2>&1 || { echo >&2"Cannot find virtualenv. Aborting."; exit 1; }
+VVER=$(virtualenv --version)
```
Comment (ssk2) [inline dcos-cli/install-dcos-cli.sh]: Let's give this a better name. `VIRTUALENV_VERSION`. No need for brevity, bits are cheap.

File: dcos-cli/install-dcos-cli.sh [hunk 2]
```diff
@@ -20,6 +20,11 @@ MARATHON_HOST=${ARGS[2]}
...
+eval VVER=`echo $VVER | sed -e "s#[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)#\1#"`
```
Comment (ssk2) [inline dcos-cli/install-dcos-cli.sh]: Wildly unnecessary.

File: dcos-cli/install-dcos-cli.sh [hunk 3]
```diff
@@ -20,6 +20,11 @@ MARATHON_HOST=${ARGS[2]}
 MARATHON_PORT=${ARGS[3]:-8080}
 UNIVERSE_URI="https://github.com/mesosphere/universe/archive/ea.zip"
```
Comment (ssk2) [inline dcos-cli/install-dcos-cli.sh]: Let's add a print statement here "Checking virtualenv version"

**Comment by jsancio** [comment]
Looks good to me! Please squash commits and merge.

**Pull request merged**